mobify/platform-scaffold
# AMP catch-up with develop

Update `amp` to the latest in `develop` + fixes required

~~**JIRA**: (link to JIRA ticket)~~

## Changes
- Merge latest changes from `develop`
- Fix stuff that was broken (PDP)
- Just look at the last commit for the actual changes I made (https://github.com/mobify/platform-scaffold/pull/791/commits/bcf1641b71bceaf06d8a33952bd4542a2d8dfb24)

## How to test-drive this PR
- Run `npm run dev`
- Check out http://localhost:3000/books.html, http://localhost:3000/potions.html
